### ISSUE TITLE
When placing or removing a block there is a instant FPS drop.

### ISSUE THREAD

**Issue by nexitem** [StrangeLoopGames/EcoIssues#6588]
Since 7.3.0 there is a FPS drop you get everytime you remove or place a block.
It seems to be caused by the new effect that comes out of blocks... the little pixels that "explode"

System Specs:
AMD FX8350
AMD RX580 8GB
16GB DDR3
Windows 10

**Comment by nexitem** [comment]
Okay it seems not to be the pixels, I was wrong about that I believe. I tested it in singleplayer and there I don't have this issue, at least not on a fresh world. I will make a testserver with the same settings I got my main world on to see if its not clientside but serverside I guess. But I can confirm that all my players have the issue, when someone else places a block there is also a short 0.5~ sec FPS drop.

**Comment by nexitem** [comment]
Its so hard to track this down, It seems to be much worse close to stockpiles... But in the woods far away from the village I got almost none of it.

I don't know how the code works but I got a feeling its connected to the room system... are you guys checking if something changes everytime a block has been placed? For example "solid ground" for furniture? All I can say is that pefromance in general got extremly bad since 7.3.0
